lib/core/src/lib/dynamic-chip-list/dynamic-chip-list.component.ts

```diff
@@ -64,6 +70,10 @@ export class DynamicChipListComponent implements OnChanges, OnInit, AfterViewIni
     @Input()
     limitChipsDisplayed = false;
 
+    /** Round up chips */
+    @Input()
+    roundedChips = false;
```

Comment:
If we want to add a control for user to round up chips this input should take just border radius value like '10px', also new input should be documented

Reply:
that was set up with Shane